christfellowshipchurch/web-app-v2
# Require auth before contacting group leader

TO TEST: 

- When logged out try to contact a leader. The auth modal should pop up. Once you are logged in you should see the rest of the contact leader flow as before.
- If you are logged in the contact a leader should work as expected and not show the auth modal.

![2021-02-23 11 04 06](https://user-images.githubusercontent.com/2528817/108879592-07848880-75c7-11eb-953f-945ed1271268.gif)
